src/components/UserDetailSections/UserAffiliations/UserAffiliations.test.js

```diff
@@ -1,16 +1,33 @@
-import { screen } from '@testing-library/react';
-
+import {
+  screen,
+  fireEvent,
```

Comment:
Could you please use `import userEvent from '@testing-library/user-event`' for the same purpose?